rpcs3/Emu/Cell/PPUInterpreter.cpp

```diff
@@ -2829,8 +2829,8 @@ bool ppu_interpreter::BC(ppu_thread& ppu, ppu_opcode_t op)
 
 	ppu.ctr -= (bo2 ^ true);
 
-	const bool ctr_ok = bo2 | ((ppu.ctr != 0) ^ bo3);
-	const bool cond_ok = bo0 | (ppu.cr[op.bi] ^ (bo1 ^ true));
+	const bool ctr_ok = bo2 || ((ppu.ctr != 0) ^ bo3);
+	const bool cond_ok = bo0 || (ppu.cr[op.bi] ^ (bo1 ^ true));
```

Comment:
MSVC generates really shitty code if you use || or && operators there and all other places, although it shouldn't affect it in theory.